README.md

```diff
@@ -127,19 +139,30 @@ interface Line {
   number: number;
   text: string;
   ending: string;
+  blocks: { [type: string]: Line[] }[];
 }
 ```
 
 ### Options
 
 ```ts
 interface Options {
-  newlines?: string[];
+  newlines?: string[]|RegExp;
+  blocks?: BlockOptions[]|BlockOptions;
 }
 ```
 
+### BlockOptions
+
+```ts
+interface BlockOptions {
+  type: string;
+  start: RegExp;
+  end: RegExp;
+}
+```
 
-### linez(file: string|Buffer): Document
+### linez(file: string|Buffer, options?: IOptions): Document
```

Comment:
I should probably explain that the reason I didn't support this was very intentional, for performance reasons. Imagine that you have 100,000 files to parse, but the options are beings passed on a per-file basis like this. A new `StringFinder` has to be created for each file parse.

My idea was that if you really needed to do that you would just create a new instance of linez and configure it to work with those file-specific line endings. This way, the `StringFinder` is always cached and ready to go.

What are your thoughts?